### PR TITLE
Issue/8

### PR DESCRIPTION
**Pull request by vanduc1102**
#### What does this PR do?
- Support open branch in stash
- Support choose a name on installing

### CODE REVIEW

File: o.sh [hunk 1]
```diff
@@ -68,4 +97,5 @@ get_open_program() {
 url=$(get_url)
 if [ "" != "$url" ]; then
 	$(get_open_program) $url
+	# echo $url
```
Comment (plutov) [inline o.sh]: Please remove this comment.

File: install.sh [hunk 2]
```diff
@@ -6,7 +6,7 @@ set -e
 
 DOWNLOAD_URL="https://raw.githubusercontent.com/plutov/o/master/o.sh"
 TMP_FILE="/tmp/o.sh"
-BIN_FILE="/usr/local/bin/o"
+BIN_FILE="/usr/local/bin/"
```
Comment (plutov) [inline install.sh]: We can remove it now, as we set it on line 31.

**Review by plutov — CHANGES_REQUESTED** [review]

File: o.sh (hunk 1 above)
Comment (vanduc1102) [reply o.sh]: Ok, l leave it for debug purpose.

File: install.sh (hunk 2 above)
Comment (vanduc1102) [reply install.sh]: Done

File: o.sh (hunk 1 above)
Comment (vanduc1102) [reply o.sh]: Done

**Pull request merged**